### PR TITLE
Update banner-notifications.mdx

### LINKED ISSUE #2366: Add Banner Notification to Design System site

### PR DESCRIPTION
**Pull request by vavalos5**
[Deploy Preview](https://deploy-preview-2363--keen-mayer-a86c8b.netlify.app/templates/global-banner-notifications)

#### What does this PR do?
This PR starts the Banner Notification page for the design system

- [x] Diagram is still a WIP 

#### Where should the reviewer start?
Banner-Notification folder
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

#### What are the relevant issues?
closes #2366

### CODE REVIEW

**Comment by taysea** [comment]
> forgot to add this link to this section. It should read as:
> 
> "Utilize writing guide to stay within character count"

what is/where is the writing guide?

File: aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js
Comment (taysea) [inline aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js:48]: we can't change these here because it's being used in NameValueList

File: aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDoExample.js
Comment (taysea) [inline aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDoExample.js:7]: remove comments? or, I think we could add `onClick` and have it display an alert that says "Clicking on a banner notification could route the user to a notification center for more details."

File: aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDontExample.js
Comment (taysea) [inline aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDontExample.js:11]: Same comment as above.

File: aries-site/src/examples/templates/banner-notifications/Examples/BannerNotificationCritical.js
Comment (taysea) [inline aries-site/src/examples/templates/banner-notifications/Examples/BannerNotificationCritical.js:11]: Same comment as above

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:20]:
```suggestion
1. [**Status Indicator**](/templates/status-indicator) - an icon made of a shape, color and content to capture the user’s attention.
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:56]:
```suggestion
- Never sticky, should scroll when there is other content.
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:78]: Need to adjust github links

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:122]: Think the Do and Don't should stack one on top of each other so give the banner a more realistic width.

File: aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDoExample.js
Comment (britt6612) [reply aries-site/src/examples/templates/banner-notifications/Dos&Donts/BannerNotificationDoExample.js:7]: that's what I was debating about if we wanted an `alert` I commented out though since that functionality isn't available through grommet.

File: aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js
Comment (britt6612) [reply aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js:39]: that would be better if we made the text longer since that box grows with the text the diagram just bases it off the middle of the box

File: aries-site/src/examples/templates/banner-notifications/Examples/BannerNotificationCritical.js
Comment (britt6612) [reply aries-site/src/examples/templates/banner-notifications/Examples/BannerNotificationCritical.js]: took it out and the `box`

File: aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js
Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js]: yes that change shouldnt be in there my mistake

File: aries-site/src/data/structures/templates/templates.js
Comment (taysea) [inline aries-site/src/data/structures/templates/templates.js:607]: Should this page be called "Global Banner Notifications"? Since all the guidance is around global ones.

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:33]:
```suggestion
Global Banners are used to showcase Warning and Critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:50]:
```suggestion
- Always located directly below the Global Header without a gap and should be implemented inline with content so no content or actions are obscured.
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:51]:
```suggestion
- Can persist across multiple sessions until dismissed if necessary.
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:54]: These should all have "." period at the end.

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:67]: It seems this section should just be under "Guidelines"? and this header is unnecessary.

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:69]: I feel like all of these should be listed under an h2 called "Variants" like we do for other pages.

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:135]: Why is this an h2? shouldn't it be under "Guidelines" as an h3

File: aries-site/src/data/structures/templates/templates.js
Comment (britt6612) [reply aries-site/src/data/structures/templates/templates.js:607]: good point I agree

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:128]: Should this be under "Guidelines"?

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/banner-notifications.mdx:144]:
```suggestion
    bestPractice={{type: 'do', message: 'Global Banners should always have an anchor (or anchors) that is relevant to resolving the notification. It is not recommended to use more than two anchors. Also, low-severity notifications should be dismissible.'}}
```

File: aries-site/src/pages/templates/banner-notifications.mdx
Comment (britt6612) [reply aries-site/src/pages/templates/banner-notifications.mdx:69]: thats fair I can rework the page to match what we have

File: aries-site/src/data/structures/templates/templates.js
Comment (taysea) [inline aries-site/src/data/structures/templates/templates.js:611]:
```suggestion
     attract user attention to system conditions and updates.`,
```
OR

```suggestion
     attract user's attention to system conditions and updates.`,
```

File: aries-site/src/data/structures/templates/templates.js
Comment (taysea) [inline aries-site/src/data/structures/templates/templates.js:624]: Same as above.

File: aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js
Comment (taysea) [inline aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js:73]: Why are we exporting this?

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:37]:
```suggestion
### Do Not Use Global Banners For
```

File: aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js
Comment (taysea) [inline aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js:68]: vertical adding/height seems too large, horizontal padding seems too small

File: aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js
Comment (britt6612) [reply aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js:73]: Im importing this for the first example of the page.

File: aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js
Comment (britt6612) [reply aries-site/src/examples/templates/banner-notifications/BannerNotificationDiagram.js:68]: Ill adjust

File: aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js
Comment (taysea) [reply aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js:73]: Can we use the new grommet component instead?

Comment (britt6612) [reply aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js:73]: good point let me switch that out

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:38]: It seems like all of these use cases are for "Toast Notifications"? But it seems like we are only saying the last bullet point is. Maybe we separate the sub-bullet into it's own paragraph? see suggestion below.

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:41]:
```suggestion

For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:44]:
```suggestion
- Do not use `onClose` when information is important and needs to remain persistent. Also, consider whether or not the information should remain persistent across multiple sessions.
```
also, I don't understand the second sentence. are we saying that if information needs to remain persistent across multiple sessions it should never be dismissible? or that we should persist a banner until a user dismisses it (meaning it could persist over multiple sessions but they can dismiss it anytime, afterwhich it will not appear?) @KennyAtHPE

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:45]:
```suggestion
- Global Banners are not clickable. Instead use anchors to direct users to take further action or to view the issue in full detail.
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:46]:
```suggestion
- Do not allow messages to exceed X characters.
```
Feels like we should be specific about a character count. What does "overly long" mean? seems subjective. @KennyAtHPE

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:56]:
```suggestion
1. **Status Indicator** is an icon made of a shape and color to capture the user’s attention
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:57]:
```suggestion
2. **Message** is a short description explaining the issue.
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:58]:
```suggestion
3. **Anchors (optional)** are used to direct users to links related to the notification.
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:59]:
```suggestion
4. **Close Button (optional)** is used to dismiss the notification.
```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [inline aries-site/src/pages/templates/global-banner-notifications.mdx:61]:
```suggestion
## Best Practices

```

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (KennyAtHPE) [reply aries-site/src/pages/templates/global-banner-notifications.mdx:44]: It'd be the latter, maybe something along the lines of "Also, global banners may remain persistent across multiple sessions until they are dismissed." would be more clear.

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (KennyAtHPE) [reply aries-site/src/pages/templates/global-banner-notifications.mdx:46]: I think we should use the same limitations we imposed before which is the 192 character limit

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (taysea) [reply aries-site/src/pages/templates/global-banner-notifications.mdx:44]: Cool then I think we should move that sentence from the "Things to Avoid" and instead just place it under guidance. @britt6612

File: aries-site/src/pages/templates/global-banner-notifications.mdx
Comment (KennyAtHPE) [reply aries-site/src/pages/templates/global-banner-notifications.mdx:41]: @taysea good suggestion. Also, did you want me to commit these suggestions or leave that to you and Brittany?

Comment (taysea) [reply aries-site/src/pages/templates/global-banner-notifications.mdx:41]: If you're good with the copy suggestion, go for it! @KennyAtHPE

**Comment by britt6612** [comment]
theme changes here https://github.com/grommet/grommet-theme-hpe/pull/241
Ill strip out what I have locally here before merging

**Comment by jcfilben** [comment]
Can we also fix the width of the don't example

Screen Shot 2022-03-11 at 1 44 24 PM

File: aries-site/src/data/structures/templates/templates.js
Comment (taysea) [inline aries-site/src/data/structures/templates/templates.js:608]:
```suggestion
    available: true,
```

**Review by taysea — APPROVED** [review]
LGTM!

**Pull request merged**
